src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,138 @@
+
+Data formats that can be imported by CockroachDB include:
+
+- [SQL](schema-design-overview.html) for the [schema import](../cockroachcloud/migrations-page.html).
+- [CSV](migrate-from-csv.html) for table data.
+- [Avro](migrate-from-avro.html) for table data.
+
+The storage media you use to export / import from can be intermediate data files or streaming data coming over the network. Options include:
+
+- Local "userdata" storage for small tables (see [`cockroach userdata`](cockroach-userfile-upload.html), [Use a Local File Server](use-a-local-file-server.html)).
+- Cloud blob storage (see [`IMPORT INTO`](import-into.html), [Import Performance Best Practices](import-performance-best-practices.html), [Use Cloud Storage](use-cloud-storage.html)).
+- Direct wire transfers (see [managed migration services](#managed-migration)).
+
+Database features for export/import from the source and target databases can include:
+
+- Tools for exporting from the source database may include: `pg_dump --schema-only` and `COPY FROM`, `mysqldump`, `expdp`, etc.
+- For import into CockroachDB, use [`COPY FROM`](copy-from.html) or [`IMPORT INTO`](import-into.html). For a bulk data migrations, most users should use `IMPORT INTO` because the tables will be offline anyway, and `IMPORT INTO` can [perform the data import much faster](import-performance-best-practices.html) than `COPY FROM`.
+
+Decide which of the options above will meet your requirements while resulting in a process that is [restartable](#restartable) and [scalable](#scalable).
+
+<a name="restartable"></a>
+
+### Design a restartable process
+
+To have a higher chance of success, design your data migration process so it can be stopped and restarted from an intermediate state at any time during the process. This will help minimize errors and avoid wasted effort.
+
+Keep the following requirements in mind as you design a restartable import/export process:
+
+- Bulk migrate data in manageable size batches for your source and target systems.
+    - This is a best practice. If something happens to the target cluster during import, the amount of wasted work will be minimized.
+- Implement progress/state keeping with process restart capabilities.
+- Make sure your export process is idempotent: the same input to your export process should return the same output data.
+- If possible, export and import the majority of your data before taking down the source database. This can ensure that you only have to deal with the incremental changes from your last import to complete the migration process.
+
+<a name="scalable"></a>
+
+### Design a scalable and performant process
+
+Once your process is [restartable and resilient to failures](#design-a-restartable-process), it's important to also make sure it will scale to the needs of your data set. The larger the data set you are migrating to CockroachDB, the more important the performance and scalability of your process will be.
+
+Keep the following requirements in mind:
```

Comment:
Could be useful to note in here that benchmarking performance will help ensure the process completes within the allotted downtime window.

Reply:
thanks! updated to add another bullet that says

> - Benchmark the performance of your migration process to help ensure it will complete within the allotted downtime window.